Projekty/Projekt1/Paulina_Jaszczuk_Adam_Frej_Jakub_Foltyn/Feature_engineering/ProjektWUM_km2 (1).ipynb

```diff
@@ -0,0 +1 @@
xqKbStxUUpmsLlvr2ZJ+69Y/mJgYOzsXbAON1XSt0IIn23rde1EdMP64D+xt97FlL/qOXPNbTFXRYUEW1TxK+BXsn2bLsHmZ7wRsq8cXIIp72laU3zf8kV6XkgubT3zHedy25bDAuxhuj82Nr0UQvhl5qQSk7sjS4HdJDVKKXPpfSaXpsPiC8uubD9LE2Um22JN3utyCdAqGRAtyy0ovsalmNcjTbnXvQ19vyw+o3IGgWxsCSFkFlC8FL1S92FtbwLF19qL7C+S72Ju9a8pfd1l1UO28aOiMZYQwnRgenyu/RDbnuVW4lzMEMKfgT9H5a4vZmhYC/wmS3YZBbgVxeM32HiakWm7cYtcFRNNxaOwOTnJxt8Y+CoV/cc55Pc1ZpYdgM2d+BybHJvhKewtpXnirTv521Q619yQ1Bp7i/kkcQ0kr0O2jUG7LMknYm8U47F2Nj5x7l2sQ7UrQ+YSSloIYWuwpetXYmbqtLWzxiHpG9j3ghdi7qmZ2Nviumx1EpP9ExvMtmlFcgjhDWwC7y6UvWjkJeA/lVh1HCewH0Wxu626yKWtV6pt1WEyLuZPyG1syrTB3pkASY2JK64TzAVOiOdIp9lO3sTmFCdXvgvbF6+m8hJwWmoKTF/sBS/T3+ZifXKvTARJXShWBiqkjL6fqwV9JtAjPg+qivsx61vG4JEZ275ZRr9dGz1er5K4/5Fc21cuY2wRIYQ1wfZZnASUMrqEEFaEEO4Cnst2PjIfm9fcLxXeH3gvhFAl1mS3yO0YMqvljsIsamCTO38l6SVstcyPscnbuTABM5NfCExOKmchhHcl/Qnbj+wGbMBthFlVOoQQzq6E3AdFC5ywRRVDsDeNzMqaFzEXyehY1t6YlWxxOqMQwkZJD2Du4wfj22Hm3FZJFwF/kW1x8Bg2qLTH3pZOx7ZGmIG9tb2HzSW8CHs7XVCJa9oZqC/pe/H/3bBVUL/AlNKTQghfS3oCu94nJI3ABrlm2ITvRiGEoSGEz2Xbr1wb52Y8itXLD4Erk3NoMkh6HhuI5mNvqD/D5ljOKUPWcdjg+pik32NvwJdjFsC7tq8ato9c2noubSsUr4CsKyTbXwHW/oYBU0IIS2Pbu13SZZhi0RObj1hECGG+pL8Dd8YpFkux6RXrKWkdvRnr83+X9AfswfmbLPEqTQjhM9n2GVdK2kzxqtVmlLa+7GwUSDo9S/ifsGfDZNnekntjL3gzQgj/jHHuxe7XNElXYor3lZhLscw6zaHvvwO0ljQwxlkZsu/j+AfgTGx3gmsx5b8j0DSEsE0r1EMIQdJ1mIXw+BDCTNmWKH+M05KexZTODkC3EELG1T8ceETSbdjcuO9j4x9U3L4qHGMl/RzzPD2O7b5wIKaE3QcQ638PolsVm7N3HNmtcYQQVkm6GRgmaQs2ZvXF+th/51JXOZFe/eC/bVuRkyX8t1jnmRWPd8U65Kr4G0Pxar1DQ5YVL4m8hC2iCECPLGUJm4P3FvZmvQJTIM/M8Roy5SZ/S4FpwHdTcU/COv0G4I3YIGeRWIGWiPuDmNcPyij3ZOxt5ktsYcVr2IrX+piCMhqzsKzHOs00oFN13/NtaB+ZOt2KWVTnYUvM26TiNsQG6IWY8rEUG1B+mIr3c2wi7FcxzkNAs3huHCVXnI3ELBlrY9lPA8eU134xpWdyTLMu1vuBqTgBOC+XvpBDHZW1anXUtrb18tpWdbeJamx/Ibar9zFlYbcYpx7mRVge6+oR4AhSYxH2AJsQ63QZNvdqNPBaqsxu2NjwVaz3YzBL2uBEnFmUXrWabRwt0c4wxf1ObM7VauCWmPbz6q7rStyD5K8rpjS/FOtoObYKc9dUHt/BtnT5ChsT+2AvuDcn4oyjcn2/EfZMWk5iBWw6nxi2b7z3q7Hx+HXgvypx/dnubb14DTMSYT8BXsaeL6tjvfw6le58bBPz9djLbL8o/3cTcUqNTzG83DEWU+KmY0rcRmwqxgiKV9f2wix7K+L5dzElLrNDQztK95t6scxPYplvAz9OyZWtzkvlVdYvU7jjVDnRatIf28piu97GHcfZuYgr/uZj8+vKdPdLOhpTqruHEJ4uK952yPEk0CCEkJ7XV2uJUx/eA84JIVRmFXutQ7a35mXAHqGOfv3FXatOlSPpIGzOwC8wl58rcY5Tw5HUD/gmZuVphrnqDsTcbsl4I7C5TJlJ+r/DLHTPsJ1I6oZZC1/Bpl8MwCxa6TlItYq4aOxTbI/Eb2FbXK3ArKd1hrjAYChmXVyPWXsvBe6pq0ocuCJX64mTgcvbS27rDlC07sIG26mY68OpY8TJ2yrjdAhV9G1ZJ698ic1JOwAbU97ENuxOz7dsiLn1WmNuvX9g7rGqGGfWYW7FoZhr8H1gYAjh4XJT1XwCNlf1m5h79Tng4mDfLa424o4CZS6aDLZAoSrZhC3WOBP7msIS7CsMv6vicmoU7lqt5ci+P1eeO+PKEMIV+ZHGqSvIvr9Z1urij0II7fInjeM4OwLZ91nLW0W/X8i+gMKpQlyRq+XEFWbZ9iDK8GkI4dN8yePUDSR1wiwz2fgqhPBmPuVxHKfqkdSO4s18s/FG2I4tsJzccEXOcRzHcRynhuIbAjuO4ziO49RQXJFzHMdxHMepobgi5ziO4ziOU0NxRc5xHMdxHKeG8v/dei5248V5+wAAAABJRU5ErkJggg==\n","text/plain":["<Figure size 720x864 with 1 Axes>"]},"metadata":{"tags":[]}}]},{"cell_type":"markdown","metadata":{"id":"l3PR9LxMDRjA"},"source":["Jak widać naiwny model bayesowski zdecydowanie odstaje od reszty.\n","Pogrupowanie danych w tym przypadku zmienia jedynie ich rozrzut, a nie ogólny wynik."]}]}
```

Comment:
<p>Doceniam, że zapisano kod w postaci funkcji - docelowo lepiej by było gdyby nazwy rodzaje modeli można było podawać jako parametry.</p><p>W momencie gdy nie wiemy jaki jest balans klas, lepiej używać innej miary niż accuracy (jednak w ramach pierwszych prototypów jest to rozwiązanie akceptowalne).</p><p> </p><p>W kolejnych eksperymentach należy pokombinować z parametrami modeli i zastosować różne miary jakości.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>